content/en/service_management/workflows/connections.md

```diff
@@ -105,6 +106,55 @@ To learn how to restrict connection use, see [Access and Authentication][4].
+
+You define the members of a connection group using a connection's _Identifier Tags_. For example, you can create a connection group consisting of AWS accounts that have the `env` tag.
+
+Each connection in the group must have a different set of tag values for the given Identifier Tags. For example:
+- `connectionA {env:staging}` and `connectionB {env:prod}` can be grouped together.
+- `connectionA {env:staging}` and `connectionC {env:staging}` can't be grouped, because the group would contain duplicate tag values.
+
+You can only add connections you have **Resolve Access** for, and you can only use connections within a group if you have Resolve Access for those connections. If you use a group with a connection to which you don't have Resolve Access _and_ when your workflow runs it points to that connection, the workflow fails with a `403 Forbidden` error. To fix this issue, you can:
+- Configure your workflow so that it can't point to a connection you don't have Resolve Access for.
+- Remove the connection you don't have Resolve Access for from the connection group.<div class="alert alert-danger"><strong>Note</strong>: If you are using a connection group for multiple workflows, removing a connection that another workflow relies on causes that workflow to fail.</div>
+
+### Create a connection group
+
+To create a connection group:
+
+1. Navigate to the [connections list][3].
+1. On the left, click **Groups**.
+1. Click **+ New Group**, then select an integration.
+1. Enter a group name, then enter a set of up to three **Identifier Tags** that the connections you want to include in your group all have.
+1. Under **Confirm Group**, use the checkboxes to select the specific members of your group.
+1. Click **Next, Confirm Access**, then choose your desired access level for the group.
+1. Click **Create**.
+
+### Use a connection group
+
+1. In your workflow, select an action that requires a connection.
+1. In the **Connection** field, in the drop-down, select the desired connection group under **Groups**.
+1. Fill in the desired values for the connection group **Identifiers**. For example, if your connection group is defined using the `env` Identifier Tag, and you have two environments, `prod` and `staging`, you could use either of those values (or an expression that evaluates to one of those values).
+1. Fill in any other required step values, then click **Save** to save your workflow.
+
+### Update a connection group
+
+If you have edit access to a connection group, you can update the following attributes:
+- Group name
+- Identifier Tags (these can never be empty, but they can be completely replaced)
+- Connections (a group can be empty)
+
+<div class="alert alert-warning"><strong>Note</strong>: After you create a connection group, you can't update the integration associated with the group.</div>
```

Comment:
I'm not sure someone will ever want to do this, so i'm not sure it's worth calling out.